aim/db/model_base.py

```diff
@@ -47,6 +50,14 @@ class HasId(object):
                    default=utils.generate_uuid)
 
 
+class HasDisplayName(object):
+    display_name = sa.Column(sa.String(256))
+
+
+class HasTenantNameKey(object):
+    tenant_name = name_column(primary_key=True)
```

Comment:
Should this be a foreign key? I thought we weren't going to support "partial trees" in AIM (as in, objects pointing missing ones).